src/main/resources/lang/default.lang

```diff
@@ -343,17 +343,31 @@ tree types:
 
 # -- Time --
 time:
-	millisecond: millisecond¦s
-	tick: tick¦s
-	second: second¦s
-	minute: minute¦s
-	hour: hour¦s
-	day: day¦s
-	week: week¦s
-	month: month¦s
-	year: year¦s
-	real: real, rl, irl
-	minecraft: mc, minecraft
+  millisecond: millisecond¦s
+  tick: tick¦s
```

Comment:
milliseconds and ticks can no longer be parsed since they have no full or short name.